### DISCUSSION TITLE
Has the hero's third ability DAWNBREAKER ever worked correctly?

### DISCUSSION THREAD

**Discussion by Vayomi** [ValveSoftware/Dota2-Gameplay#15581] [General]
Ability name
LUMINOSITY

Description
Ability Description:
After 3 attacks, the hero is filled with power, causing his next attack to deal critical damage and heal all allied heroes within a 650 radius for 35%/40%/45%/50% of the damage. Healing from an attack against a creep or neutral creature will be weaker.
And additional information by pressing Alt:
Healing from this ability is considered vampirism and is increased by appropriate sources.

At the same time, the ability is not affected by the increase in vampirism from Senge, the increase in healing from Holy Locket, or the passive greater healing of the neutral item Paladin Sword. All these items will not in any way affect the recovery from Luminosity of both yourself and your allies.
The only thing that will affect the healing is the amount of crit damage dealt and the number of creeps and heroes hit at that moment.
Luminosity does not work correctly, since no known enhancements work for it. I hope for a speedy fix.

Even when activating the Revenants Brooch item, treatment from Luminosity does not work at all, both on yourself and on allies. (What is noteworthy is that when you use normal attacks with an active brooch, you receive treatment from magical vampirism, which is given to you by the Brooch item. But when you use Starbreaker’s first ability with an active Brooch, magical vampirism also disappears. Somehow it works too crookedly together. )

I still don’t understand why Luminosity doesn’t work on illusions, when most other heroes have passives that work on illusions.

What do you think? Is this a bug or a feature?

**Comment by NicholasSmith56892** [comment]
Based on the description provided, it appears that there may be several issues with the functionality of the "Luminosity" ability in the game. Here are some observations and thoughts.

Inconsistencies with item interactions: It seems that certain items, such as Senge, Holy Locket, and Paladin Sword, are not affecting the healing provided by Luminosity as expected. Additionally, the Revenants Brooch item does not seem to interact properly with Luminosity. These inconsistencies could be indicative of a [BallSportsGear](https://ballsportsgear.com/) bug or unintended behavior.

Illusion interaction: The fact that Luminosity does not work on illusions while many other hero passives do raises questions about consistency in-game mechanics. It's worth considering whether this is intentional or if it should be addressed to align with other similar abilities.

Interaction with other abilities: The interaction between Luminosity and other abilities, such as Starbreaker's first ability with an active Brooch, seems to be causing issues. This could be due to conflicting mechanics or unintended interactions between abilities and items.

Overall, based on the description provided, it appears that there are indeed issues with the functionality of the Luminosity ability. Whether these issues are classified as bugs or intentional design choices may require further investigation by the game developers. Players experiencing these issues may want to report them to the game's support or forums for review and potential resolution in future updates.
